directories.js

```diff
@@ -22,6 +19,20 @@ platform.win32 = {
   ],
 }
 
+if (process.platform === 'win32') {
+  if (process.env.USERPROFILE) {
+    platform.win32.appPath.push(path.join(process.env.USERPROFILE, 'Desktop'))
+  }
+
+  if (process.env.APPDATA) {
+    platform.win32.appPath.push(path.join(process.env.APPDATA, 'Microsoft', 'Windows', 'Start Menu', 'Programs'))
+  }
+
+  if (process.env.ProgramData) {
+    platform.win32.appPath.push(path.join(process.env.ProgramData, 'Microsoft', 'Windows', 'Start Menu', 'Programs'))
```

Comment:
Are you sure these environment variables exist when launching Zazu from the exe?